app/src/middleware/validation.js

```diff
@@ -0,0 +1,27 @@
+const config = require('config');
+const Problem = require('api-problem');
+const { validators } = require('../components/validators');
+
+const validateEmail = async (req, res, next) => {
+  const errors = await validators.email(req.body, config.get('server.attachmentLimit'));
+  if (errors.length > 0) {
+    return new Problem(422, {
```

Comment:
https://stackoverflow.com/questions/16133923/400-vs-422-response-to-post-of-data
Currently seems to be that user entered bad data in valid JSON _can_ be supported by the current spec for **400** if I'm reading right.
Thoughts @parc-jason ?

Reply:
Sort of.  400 is a very grey area about that - why aren't you processing the call, the JSON is valid, so the server can handle it, but the business can't because content within the JSON is not acceptable (thus a 422). Plus it makes it a lot clearer to the caller that they are handling validation errors and not another type of Bad Request.